src/App.js

```diff
@@ -1,25 +1,34 @@
-import FindMember from './FindMember';
-import Person from './Person';
-import './App.css';
+import FindMember from "./components/findMember/FindMember";
+import Person from "./components/person/Person";
+import "./App.css";
 
 function App() {
+  const persons = [
+    { name: "Carolien Bloeme", isChecked: false },
+    { name: "Sun Jun", isChecked: true },
+    { name: "Song Bao", isChecked: false },
+    { name: "Olivia Arribas", isChecked: true },
+    { name: "BonginkosiMdladlana", isChecked: false },
+    { name: "Arina Belomestnykh", isChecked: true },
+    { name: "Jacqueline Likoki", isChecked: true },
+  ];
   return (
-    <div className="App">
-      <div className="div1">
-        <div className="div2">
-          <h2>Add members to Front-end development team</h2>
-          <FindMember />
-          <Person name="Carolien Bloeme" isChecked={false} />
-          <Person name="Sun Jun" isChecked={true} />
-          <Person name="Song Bao" isChecked={false} />
-          <Person name="Olivia Arribas" isChecked={true} />
-          <Person name={"Bonginkosi Mdladlana"} isChecked={false} />
-          <Person name="Arina Belomestnykh" isChecked={true} />
-          <Person name="Jacqueline Likoki" isChecked={true} />
-        </div>
-      </div>
+    <div className="main-contianer App">
+      <h2>Add members to Front-end development team</h2>
+      <FindMember />
+      {
+        persons.map((person)=>(<Person name={person.name} isChecked={person.isChecked}/>))
```

Comment:
Fix this error and let me know why you got it
<img width="1456" alt="image" src="https://github.com/user-attachments/assets/b46ca243-feb4-429a-827c-1025133185cf">